### ISSUE TITLE
can't install with remix version 0.21.0

### ISSUE THREAD

**Issue by AlonGvili** [sergiodxa/remix-auth#48]
```
❯ npm install --save remix-auth
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: remix-pesonal-cms@undefined
npm ERR! Found: @remix-run/react@0.21.0
npm ERR! node_modules/@remix-run/react
npm ERR!   @remix-run/react@"^0.21.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @remix-run/react@"^0.20.1" from remix-auth@2.2.0
npm ERR! node_modules/remix-auth
npm ERR!   remix-auth@"^2.2.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\PoshI\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\PoshI\AppData\Local\npm-cache\_logs\2021-11-18T07_54_37_799Z-debug.log
```

**Comment by sergiodxa** [comment]
@AlonGvili this is released on v2.3.0

**Comment by AlonGvili** [comment]
> @AlonGvili this is released on v2.3.0

thanks bro.
